tests/Action/LoginActionTest.php

```diff
@@ -228,7 +228,8 @@ private function getAction(): LoginAction
             $this->templateRegistry,
             $this->tokenStorage,
             $this->translator,
-            $this->csrfTokenManager
+            $this->csrfTokenManager,
+            true,
```

Comment:
@VincentLanglet if i want to make a mock test case for resetting false, should i make a whole new LoginAction(withoutReset)Test class or should i try to extend this one?

Reply:
Extending doesn't seem a bad idea in this case if it's possible

Reply:
i extended the Test Data Provider with Resetting=false